sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregate.scala

```diff
@@ -39,7 +40,9 @@ case class TungstenAggregate(
 
   override private[sql] lazy val metrics = Map(
     "numInputRows" -> SQLMetrics.createLongMetric(sparkContext, "number of input rows"),
-    "numOutputRows" -> SQLMetrics.createLongMetric(sparkContext, "number of output rows"))
+    "numOutputRows" -> SQLMetrics.createLongMetric(sparkContext, "number of output rows"),
+    "numBytesUsed" ->
```

Comment:
+1 `peakMemoryUsed`. We will show the unit in the value string, right? So, we do not say the unit in the metric name?